source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="chap-Renewing_certificates_{context}"]
+= Renewing certificates before they expire
+
+[role="_abstract"]
+{virt-product-fullname} certificates now follow a standard 398 day lifetime and *must* be renewed once per year.
+
+[WARNING]
+====
+Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
```

Comment:
We do not use this markup for "NOT" or all caps. Please change to "Do not let certificates expire."

"error prone" and "time consuming" should be hyphenated because they are compound adjectives.